model/utils.js

```diff
@@ -60,3 +61,66 @@ export function useLocalStorage(key, initialValue) {
 
   return [storedValue, setValue];
 }
+
+export const WebMonetizationContext = createContext();
+
+export const WebMonetizationProvider = ({ children }) => {
```

Comment:
This ended up pretty well, I think. Seems worth it to factor out into a library others can use too after MozFest, maybe in one of those follow up posts.